Common/Core/PID/PIDTOF.cxx

```diff
@@ -17,6 +17,15 @@
 ///
 
 #include "PIDTOF.h"
+
+// O2Physics includes
+#include "Common/Core/CollisionTypeHelper.h"
+
+// O2 includes
+#include "DataFormatsParameters/GRPLHCIFData.h"
+#include "Framework/Logger.h"
```

Comment:
Please do not break the includes with comments. It prevents the automatic sorting.
Also include external headers with `<>`.

Reply:
> Hi @vkucera no, I would like to keep the comments

Will you manually keep the correct order?

Reply:
> I think I added the `<>` includes, I enabled the verify of the commits to avoid missing them and it works fairly nice, I think I added the `<map>` below

I meant the `""` vs `<>` distinction. Anything outside O2Physics should be included with `<>`.